config/manifests/bases/trustee-operator.clusterserviceversion.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
     alm-examples: '[]'
     capabilities: Basic Install
     categories: Security
-    containerImage: quay.io/confidential-containers/trustee-operator:v0.1.0
+    containerImage: quay.io/confidential-containers/trustee-operator:v0.2.0
     operatorframework.io/suggested-namespace: trustee-operator-system
     support: Confidential Containers Community
   name: trustee-operator.v0.0.0
```

Comment:
I'm wondering if the name should reflect the version correctly as well? 
eg trustee-operator.v0.2.0